packages/lisk-transactions/fixtures/transactions.json

```diff
@@ -1122,36 +1097,11 @@
 			"aa609c506bf96cd4c179cff4dacae3574ee84a075adc8b626181d7a357fa30498ac04ee4a536848674131a7629871c7e89c3d7cf5b98fe8bb0c84e7144da890f",
 		"id": "13972423096514804371"
 	},
-	{
-		"type": 4,
-		"amount": "0",
-		"fee": "3000000000",
-		"recipientId": null,
-		"senderPublicKey":
-			"c094ebee7ec0c50ebee32918655e089f6e1a604b83bcaa760293c61e0f18ab6f",
-		"timestamp": 54196080,
-		"asset": {
-			"multisignature": {
-				"min": 0,
-				"lifetime": 18,
-				"keysgroup": [
-					"+740834a59435d283fd3fb30ad5d7cbde2550e82471b73abedcffb61eaa6298e4",
-					"+9bc8972fb01b70eb4624df5d4d4c7c00a51fd73958c50efaefe55260889aedd6",
-					"+66a68de8047bbe788f5ec5fbae6baf84c6438606f4e6fdf91b791113a0506ea6",
-					"+5f7c4b9b6f976a400dba8d0cc7f904603ea4ffe1d8702c80576a396037c49970",
-					"+4e4a6b5cf7b8840ba521dfae5914f55ec3805c7d5cf25dfbf44fac57f9c5f183"
-				]
-			}
-		},
-		"signature":
-			"57b54da646c7567df86fec60aa57a40bfadb6cdc65cccecfc442c822a7b0372f4958a280edd3fc2d83d38e2d3bf922a1da01249c500f0309a9638e941a21c501",
-		"id": "13916871066741078807"
-	},
```

Comment:
Why don't we move these to a fixtures file for transactions which were previously valid but now aren't?

Reply:
Addressed.